### PR TITLE
Input parsing

### PR DESCRIPTION
**Pull request by Dajamante**
Parser for standard-formated input files:

- int numpoints
- list points
- double length1
- double radius1
- double epsilon
- double area1
- int Q_PTS
- int quads
- double dist
- int N_PTS
- int K_PTS
- int A_PTS
- int B_PTS
- int C_PTS
- int D_PTS
- int E_PTS
- int F_PTS
- int G_PTS
- double length2
- double radius2
- double area2
- list LCM
- list PUV

### CODE REVIEW

File: Assignment1/generate_datapoints.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# double area2
+# list LCM
+# list PUV
+
+
+import random
+import pickle
+
+
+def generate_datatpoints(x):
```
Comment (ztick) [inline Assignment1/generate_datapoints.py:30]: Please include documentation for each function (short comment explaining what it does, if you're using VScode I recommend the extension "Python Docstring Generator", I just downloaded it and it's really really easy)

File: Assignment1/generate_datapoints.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# T/F vector
+
+
+def generate_puv():
+    puv = [random.randint(0, 1) for _ in range(15)]
+    data_file = open('./datapoints.dat', 'wb')
+    pickle.dump(puv, data_file)
+    # data_file.close()
+
+    # x = number of points
```
Comment (ztick) [inline Assignment1/generate_datapoints.py:60]: should these comments be here?

**Review by ztick — CHANGES_REQUESTED** [review]
Very nice work guys, well done!
We just need some documentation and it's ready to merge imo :)

**Review by ztick — APPROVED** [review]
LGTM

**Review by gmm97 — APPROVED** [review]
1) Added assertions for each input in line with conditions given in the problem statement
2) Modified input txt file to format param as NAME=VALUE for better readability
3) Changed output write format to .txt so the formats are consistent for both reading and writing params to file

**Review by ztick — APPROVED** [review]
Nice! You're right about the readability, and the asserts are great functionality.

**Pull request merged**